easypermissions/src/main/java/pub/devrel/easypermissions/EasyPermissions.java

```diff
@@ -42,6 +42,8 @@
  */
 public class EasyPermissions {
 
+    public static final int SETTINGS_REQ_CODE = 7;
```

Comment:
While it doesn't make a difference, let's chance this to something less likely to intersect with other libs, how about `16061`.

Chosen via this method: https://xkcd.com/221/ :-)